File: bench/splice/fixtures/task4/textutils_test.go
package textutils

import "testing"

func TestFirstWordNormal(t *testing.T) {
	if got := FirstWord("hi there"); got != "hi" {
		t.Errorf("FirstWord(%q) = %q, want %q", "hi there", got, "hi")
	}
}

func TestFirstWordEmpty(t *testing.T) {
	if got := FirstWord(""); got != "" {
		t.Errorf("FirstWord(%q) = %q, want %q", "", got, "")
	}
}

File: bench/splice/fixtures/task4/text_utils.test.js
// Tests for text_utils. The empty input test fails until the task is done.
const test = require("node:test");
const assert = require("node:assert");
const { firstWord } = require("./text_utils");

test("first word of normal text", () => {
  assert.strictEqual(firstWord("hi there"), "hi");
});

test("first word of empty text", () => {
  assert.strictEqual(firstWord(""), "");
});
